examples/example-1-count-to-10/page.js

```diff
@@ -16,11 +16,15 @@ const numbers = [
   "ten,10"].join('\n');
 
 let resolve, reject;
-grist.rpc.registerFunc('getImportSource', () => new Promise((_resolve, _reject) => {
-  console.log("saving resolver", _resolve);
-  resolve = _resolve;
-  reject = _reject;
-}));
+const importer = {
+  getImportSource: () => new Promise((_resolve, _reject) => {
+    console.log("saving resolver", _resolve);
+    resolve = _resolve;
+    reject = _reject;
+  })
+};
+grist.rpc.registerImpl('count_to_10', importer);
+grist.rpc.registerImpl('inline_count_to_10', importer);
 
```

Comment:
This feels awkward somehow, with promises making the code more complicated rather than simplifying it.  Still, I think it is basically sound enough.

Reply:
There could an helper func that takes care or creating promise and resolvers and hide them from the scope of the script. 

Something like in: https://github.com/gristlabs/grist-plugin-examples/compare/handle-importer#diff-3cacdd143267c2d6bac6c88e54b5b6e2 

Do you think it would be useful ?